### ISSUE TITLE
Trade companion does not work in Japanese

### ISSUE THREAD

**Issue by Rigaln** [PoE-Overlay-Community/PoE-Overlay-Community-Fork#314]
# 🐞 bug report
Trade chats are not added to the overlay even if they are received in Japanese.
Fake in settings was working.

**Expected behavior**
It adds as shown in the demo video

## 🌍  Your Environment

- OS: win10
- Version: Setup-0.8.21
- PoE: Steam 3.20.0b Japanese (language)

**Comment by WhiteFang5** [comment]
You're right about the trades not being picked up. Unfortunately I don't speak Japanese to compose the regexes and add them in.

**Comment by melink14** [comment]
How hard would it be for a Japanese speaker to add those regexes? Is it just a matter of adding to a pre-existing list and testing?

**Comment by wratho** [comment]
@WhiteFang5 does 0.8.21 fix this?

**Comment by Rigaln** [comment]
Overwolf's poe overlay supports Japanese, but it's difficult?
I don't want to use Overwolf's poe overlay because it freezes the discord!

**Comment by Rigaln** [comment]
I changed the language for whispers to English, and when the whispers actually flew in English, they did not respond!
Is it because the item name is in Japanese?
